### PR TITLE
Missing tests

### PR DESCRIPTION
**Pull request by DeboraBSilva**
Add missing tests for components and functions.

### CODE REVIEW

File: src/__tests__/Deck.test.js [hunk 1]
```diff
@@ -0,0 +1,15 @@
+import { render, screen } from "@testing-library/react";
+import Deck from "../components/Deck";
+
+describe("Deck Component", () => {
+  test("renders all cards received through an array", () => {
```
Comment (wenderjean) [inline src/__tests__/Deck.test.js]: Seria bom testar os edge cases tambem, exemplo, o que acontece se o componente receber um array vazio?

File: src/__tests__/rows.test.js [hunk 2]
```diff
@@ -0,0 +1,74 @@
...
+          [4, 7, 1],
+          [5, 8, 2],
+        ]);
+      });
+    });
+  });
+});
+
+describe("#setRows", () => {
+  describe("when deck is invalid", () => {
```
Comment (wenderjean) [inline src/__tests__/rows.test.js:45]: Acho que esse teste não deveria estar aqui, ele testa o `reorderRows`.

File: src/__tests__/App.test.js [hunk 3]
```diff
@@ -1,8 +1,10 @@
-import { render, screen } from '@testing-library/react';
-import App from '../App';
+import { render, screen } from "@testing-library/react";
+import App from "../App";
 
-test('renders 21 cards', () => {
-  render(<App />);
-  const numberOfCards = screen.getAllByRole('card').length;
-  expect(numberOfCards).toBe(21);
+describe("App Component", () => {
```
Comment (wenderjean) [inline src/__tests__/App.test.js:5]: Daria pra testar esse componente fazendo os cliques e verificando o resultado.
Tem alguns exemplos nesse link:
https://www.freecodecamp.org/news/react-testing-library-tutorial-javascript-example-code/

Comment (wenderjean) [reply src/__tests__/App.test.js:5]: Um caminho é dae uma pesquisada por "react-testing-library testing events"
